src/operations/command.ts

```diff
@@ -8,16 +8,26 @@ import { MongoError } from '../error';
 import Logger = require('../logger');
 
 import type { Server } from '../sdam/server';
+import type { Collection } from '../collection';
+import type { MongoClient, Db } from '..';
 
 const SUPPORTS_WRITE_CONCERN_AND_COLLATION = 5;
 
-interface CommandOperationOptions {
+export interface CommandOperationOptions extends OperationBaseOptions {
```

Comment:
let's call these `CommandOptions` and figure out if we still need one of those for the wire protocol classes, or if they are actually the same as these ones.